### ISSUE TITLE
Add sitrep functions to check if appropriate packages are installed

### ISSUE THREAD

**Issue by njtierney** [greta-dev/greta#441]
As stated here: https://ropensci.org/blog/2021/09/30/ropensci-news-digest-september-2021/#package-development-corner by Maëlle:

> ...[consider] adding some sort of sitrep (situation report) function...the devtools package has devtools::dev_sitrep() e.g. reports on the package development situation, with clear hints given if something is not quite right. The usethis package has usethis::git_sitrep() (using the gert package under the hood!), blogdown has blogdown::check_site(). Good candidates for checks are common pain points, so finding them might require some sort of external perspective on your package.

This could check if the Python packages are installed, perhaps:

```
greta_sitrep()
# or
check_greta_deps()
```

**Comment by njtierney** [comment]
Either in this function, or a related function, it would be great to have a way to load/check the python dependencies, so that we can have a more descriptive step to check that everything.

e.g.,

```r
greta_sitrep()
# provides nice details on what is installed, where, and what steps might need to be taken to fix installation steps?
```

```r
greta_load()
# under the hood runs `greta::check_tf_version("message")`
```

**Comment by njtierney** [comment]
closed in #476
